hash_to_array.rb

```diff
@@ -0,0 +1,12 @@
+def hash_to_array
+  hash = Hash.new
+  5.times do
+    print "Enter key: "
+    key_from_user = gets.chomp
+    print "Enter its value: "
+    value_from_user = gets.chomp
+    hash.store(key_from_user, value_from_user)
+  end
+  puts "#{hash.keys}"
+  puts "#{hash.values}"
+end
```

Comment:
nice job. dont forget to call the function at the end of the program.

another way to initialize a hash is by doing ```user_hash = {}```
I would also put another ```puts`` statement in your main function that outputs the hash itself.
so for example: ```puts hash```
good use of the store method for a hash.
something else you can do to append a key/value pair to a hash is to do
```hash["key"] = value```